### DISCUSSION TITLE
Workflow

### DISCUSSION THREAD

**Discussion by GeekyGhost** [comfyanonymous/ComfyUI#1645] [Show and tell]
![Screenshot 2023-09-30 221404](https://github.com/comfyanonymous/ComfyUI/assets/111990299/6d846e51-f8ec-4378-a58a-3795e7242536)

![ComfyUI_00647_](https://github.com/comfyanonymous/ComfyUI/assets/111990299/c25fe151-1ced-4b65-aac4-b541cf43ab66)

I love this UI so much lol. I've made my general purpose workflow now to work on my AnimateDiff one.

**Comment by rvion** [comment]
Hey ! have you checked https://github.com/rvion/cushystudio ? wanna join early access and give your workflow some superpowers ?
